.github/workflows/main.yml

```diff
@@ -19,21 +19,23 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: ['3.10'] # Only supported one at the moment
```

Comment:
Actually that was true for kernelci-api, but I believe kernelci-core might work with earlier versions.  So it's worth trying with 3.7 and 3.10 to see what happens.

Reply:
I have checked with python version 3.7, 3.8, 3.9, and 3.10
https://github.com/kernelci/kernelci-core/actions/runs/1626891867
It is working with all the versions.